README.md

```diff
@@ -1,951 +1,236 @@
-    .code("8870af1372ada7a18fdff4fa4ca1a60f4d542272")
-    .redirectUri("http://localhost/moip-sdk/callback")
-    .grantType(GrantType.AUTHORIZATION_CODE)
-);
-System.out.println(connect);
-```
+### Other examples
+If you want to see another functional examples, check this folder. [:file_folder:](https://github.com/moip/moip-sdk-java/tree/master/src/test/java/br/com/moip/examples/)
 
-### Atualizar token OAuth
-```java
-Connect connect = api.connect().authorize(new ConnectRequest()
-    .refreshToken("80ca5fb244674117be068d2535ecbe2f_v2")
-    .grantType(GrantType.refresh_token)
-);
-System.out.println(connect);
-```
-
-## Tratamento de Exceções
+## Exceptions treatment
+| errors | cause | status |
+| :---: | :---: | :---: |
+| UnautorizedException | to authentication errors | == 401 |
+| ValidationException | to validation errors | >= 400 && <= 499 (except 401) |
+| UnexpectedException | to unexpected errors | >= 500 |
 
-Quando ocorre algum erro na API, é lançada a exceção UnexpectedException para erros inesperados e ValidationException 
-para erros de validação.
+> :warning: To catch these errors, use the bellow treatment:
 
 ```java
 try {
  Payment createdPayment = api.payment().create(
         //...
     );
+} catch(UnauthorizedException e) {
+  // StatusCode == 401
 } catch(UnexpectedException e) {
-  //StatusCode >= 500
+  // StatusCode >= 500
 } catch(ValidationException e) {
-  //StatusCode entre 400 e 499 (exceto 401)
+  // StatusCode entre 400 e 499 (exceto 401)
 }
 ```
 
-## Documentação
-
-[Documentação oficial](https://moip.com.br/referencia-api/)
+## Moip documentation
 
-## Licença
+### Docs
+To stay update about the **Moip Products**, check the [documentation](https://dev.moip.com.br/v2.0/docs).
 
-[The MIT License](https://github.com/moip/moip-sdk-java/blob/master/LICENSE)
+### References
+Read more about the **Moip APIs** in [API reference](https://dev.moip.com.br/v2.0/reference).
 
-## Comunidade Slack [![Slack](https://user-images.githubusercontent.com/4432322/37355972-ba0e9f32-26c3-11e8-93d3-39917eb24109.png)](https://slackin-cqtchmfquq.now.sh)
+## Getting help
+We offer many ways to contact us, so if you have a question, do not hesitate, talk to us what you need. For questions about API or business rules, contact us by [support](https://dev.moip.com.br/v2.0/) or [[slack](#Slack)](https://slackin-cqtchmfquq.now.sh/). But, if you have a question or suggestion about the SDK, feel free to open an **issue** or **pull request**.
 
-Tem dúvidas? Fale com a gente no [Slack](https://slackin-cqtchmfquq.now.sh/)! 
\ No newline at end of file
+## Contributing
+Do you have an enhancement suggest or find something to fix? Go ahead, help us and let your mark on Moip, open **pull requests** and **issues** against this project. If you want to do it, before, please read the `CONTRIBUTING.md` to be sure everyone follows the same structure and planning of the project. Remember, we :heart: contributions. :rocket:
```

Comment:
- or **found** something to fix?
If you want to do it, ~~before~~,  please read (tira esse "before", ficou estranho)